hack/lib/install.sh

```diff
@@ -86,3 +94,15 @@ verify_docker_installed(){
   }
 }
 
+# install CNI plugins
+function install_cni_plugins() {
+  # install CNI plugins if not exist
+  if [ ! -f "/opt/cni/bin/loopback" ]; then
+    echo -e "install CNI plugins..."
+    mkdir -p /opt/cni/bin
+    wget https://github.com/containernetworking/plugins/releases/download/v1.1.1/cni-plugins-linux-amd64-v1.1.1.tgz
+    tar Cxzvf /opt/cni/bin cni-plugins-linux-amd64-v1.1.1.tgz
+  else
+    echo "CNI plugins already installed and no need to install"
+  fi
```

Comment:
`containerd` can't run without installing the CNI plugin?

Reply:
if we don't install CNI plugins first, it cannot run or start pod totally and will report error
```
failed to setup network for sandbox "xxxxxxxxxxxxxx": failed to find plugin "loopback" in path [/opt/cni/bin]
```